dss/storage/bundles.py

```diff
@@ -119,3 +120,45 @@ def _latest_version_from_object_names(object_names: typing.Iterator[str]) -> str
             version = current_version
 
     return version
+
+
+def enumerate_avaliable_bundles(replica: str = None,
```

Comment:
I'd prefer the contents of this method to be inlined into `dss.bundles.__init__.py enumerate`. You can consider a nested method.

This can be handled in a future PR.

Reply:
addressed

Reply:
had to revert this, it was breaking unit tests.